views/default/create_packages.pdt

```diff
@@ -0,0 +1,465 @@
+        <?php
```

Comment:
Not sure why, but when I submit this page I get an error about an invalid package group ID.  Does that happen to you?